### PR TITLE
console docs review

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Annual Console docs review and clean up! Related to https://github.com/camunda/developer-experience/issues/63.

CC @ultraschuppi 

@jwulf mind having a second look over this section?

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/console/manage-clusters/create-cluster-include.md [hunk 1]
```diff
@@ -1,29 +1,24 @@
...
-1. After you've made your selection and created the cluster, view the new entry in the **Clusters** tab:
-
-![cluster-creating](./img/cluster-overview-new-cluster-creating.png)
-
-4. The cluster is now being set up. During this phase, its state is **Creating**. After one or two minutes, the cluster is ready for use and changes its state to **Healthy**:
-
-![cluster-healthy](./img/cluster-overview-new-cluster-healthy.png)
-
-5. After the cluster is created, click on the cluster name to visit the cluster detail page.
+<!--- I updated this, but do we need it anymore? Can this be removed? -->
```
Comment (akeller) [inline docs/components/console/manage-clusters/create-cluster-include.md:24]: Is this comment related to what you removed or the component in general?

File: docs/components/console/manage-clusters/delete-cluster.md [hunk 2]
```diff
@@ -8,10 +8,6 @@ description: "Follow these step-by-step instructions to remove your cluster perm
...
 
-A cluster can be deleted at any time. To delete your cluster, follow the steps below:
+A cluster can be deleted at any time. To delete your cluster, navigate to the **Clusters** tab in the top navigation and click **Delete** to the far right of the cluster name.
 
-1. Open the cluster details by clicking on the cluster name.
-2. Select the three vertical dots next to the cluster name near the top of the page to open the cluster's menu.
-3. Click **Delete**.
-
-![cluster-delete](./img/cluster-delete.png)
+<!--- Screenshot outdated -->
```
Comment (akeller) [inline docs/components/console/manage-clusters/delete-cluster.md:13]: I didn't see this in https://github.com/camunda/camunda-docs/pull/2757/files so this should be updated in the scope of this PR.

File: docs/components/console/manage-clusters/create-cluster-include.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/console/manage-clusters/create-cluster-include.md:24]: This page entirely 😬

File: docs/components/console/introduction-to-console.md
Comment (akeller) [inline docs/components/console/introduction-to-console.md:1]: "Camunda Console is the management application for the included products."

Is there a better definition or summary you can get from PM?

File: docs/components/console/console-troubleshooting/common-pitfalls.md [hunk 3]
```diff
@@ -4,16 +4,16 @@ title: Common pitfalls
...
 
 ## I cannot connect to Zeebe
 
 - Check if your [API client](../manage-clusters/manage-api-clients.md) has the necessary rights. To interact with Zeebe, the **Scope** `Zeebe` must be set.
 - Check if your credentials are configured correctly. There is a CLI tool that allows you to check the status: [`zbctl`](https://www.npmjs.com/package/zbctl). With the command `zbctl status`, you can read the topology. If this command works, the connection can be established.
-- Check if your cluster is **Healthy**: A Zeebe cluster may be temporarily unavailable. To check if your cluster is healthy, navigate to the cluster list. Click on the cluster to view its details for a closer view of the status over all products (Zeebe, Operate, Tasklist, Optimize).
+- Check if your cluster is **Healthy**: A Zeebe cluster may be temporarily unavailable. To check if your cluster is healthy, navigate to the **Clusters** tab in the top navigation. Click on the cluster to view its details for a closer view of the status over all products (Zeebe, Operate, Tasklist, Optimize).
```
Comment (akeller) [inline docs/components/console/console-troubleshooting/common-pitfalls.md:19]: "over all products".

This should be "components".

File: docs/components/console/manage-organization/switch-organization.md [hunk 4]
```diff
@@ -5,4 +5,6 @@ title: Switch organization
 
 If a user is assigned to more than one organization, the organization can be changed in the menu of the navigation bar.
 
+<!--- By clicking Manage?--->
```
Comment (akeller) [inline docs/components/console/manage-organization/switch-organization.md:8]: +1, this needs more clarification and a high-quality text description of the screenshot.

File: docs/components/console/manage-clusters/manage-api-clients.md [hunk 5]
```diff
@@ -8,6 +8,8 @@ To interact with Zeebe in the cloud from the outside, every client application m
 
 ![auth-flow](./img/client-auth.png)
 
+The application authenticates itself with Auth service using Client Id and Client Secret, Auth service validates this information and returns an access token, and the application can then use this access token to interact with Zeebe in the cloud.
```
Comment (akeller) [inline docs/components/console/manage-clusters/manage-api-clients.md:11]:
```suggestion
The application authenticates itself with Auth service using `Client Id` and `Client Secret`, Auth service validates this information and returns an access token, and the application can then use this access token to interact with Zeebe in the cloud.
```
Should "Auth" be lowercase here?

**Review by akeller — CHANGES_REQUESTED** [review]
In the "Manage organization" section, do any of these features or pages need to be grouped by role? Meaning, are some only available to owners and admins? 

Similar in "Manage clusters" - should also users expect to see this functionality or is this based on role?

If so, we could consider sections in the sidebar for role based groupings. Otherwise, let's make sure it's clear on each page who should be able to see what and if they don't have it, who they need to contact.

File: docs/components/console/introduction-to-console.md
Comment (jprcoliveira) [reply docs/components/console/introduction-to-console.md:1]: @felix-mueller any ideas?

File: docs/components/console/manage-clusters/manage-api-clients.md (hunk 5 above)
Comment (christinaausley) [reply docs/components/console/manage-clusters/manage-api-clients.md:11]: I wonder if this should actually be OAuth throughout, or am I misunderstanding?

File: docs/components/console/manage-clusters/create-cluster-include.md (hunk 1 above)
Comment (akeller) [reply docs/components/console/manage-clusters/create-cluster-include.md:24]: This page is still used by `create-cluster.md` - https://docs.camunda.io/docs/components/console/manage-clusters/create-cluster/.

File: docs/components/console/manage-clusters/delete-cluster.md (hunk 2 above)
Comment (christinaausley) [reply docs/components/console/manage-clusters/delete-cluster.md:13]: @jprcoliveira can you assist here?

**Comment by christinaausley** [comment]
Looping in @jprcoliveira for assistance with these lingering comments as @ultraschuppi is OOO this week.

**Comment by christinaausley** [comment]
> In the "Manage organization" section, do any of these features or pages need to be grouped by role? Meaning, are some only available to owners and admins?
> 
> Similar in "Manage clusters" - should also users expect to see this functionality or is this based on role?
> 
> If so, we could consider sections in the sidebar for role based groupings. Otherwise, let's make sure it's clear on each page who should be able to see what and if they don't have it, who they need to contact.

@jprcoliveira Can you confirm the following?

- [ ] View organization activity -- **all users?**
- [ ] Usage history -- **all users?**
- [ ] Usage alerts -- **owners and admin?**
- [ ] Advanced search -- **all users?**
- [ ] Switch organization -- **all users?**
- [ ] Connect your IDP with Camunda -- **not sure here**
- [ ] Delete your Camunda account -- **owner, analyst, admin, and task user?**

File: docs/components/console/manage-clusters/create-cluster-include.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/console/manage-clusters/create-cluster-include.md:24]: @ultraschuppi Do you think we can remove this page, and just add the create-cluster React component we currently have within guides to https://docs.camunda.io/docs/components/console/manage-clusters/create-cluster/?

File: docs/components/console/manage-organization/switch-organization.md (hunk 4 above)
Comment (christinaausley) [reply docs/components/console/manage-organization/switch-organization.md:8]: Resolved in latest commit.

File: docs/components/console/introduction-to-console.md
Comment (christinaausley) [reply docs/components/console/introduction-to-console.md:1]: Looping in @theburi per Felix's guidance.

File: docs/components/console/manage-clusters/manage-api-clients.md (hunk 5 above)
Comment (ultraschuppi) [reply docs/components/console/manage-clusters/manage-api-clients.md]: Oauth makes sense!

Comment (christinaausley) [reply docs/components/console/manage-clusters/manage-api-clients.md]: Resolved in latest commit🤠

**Comment by ultraschuppi** [comment]
when refering to orgRoles/orgPermissions please have a look at the code here: https://github.com/camunda-cloud/camunda-cloud-management-apps/blob/a6686ab571885e757c64a82fa036387d4816ad85/packages/shared-types/src/types/OrganizationPermission.type.ts#L78

> * View organization activity -- **all users?**

every user with a orgRole including `org.activity.read` can do that

> * [ ]  Usage history -- **all users?**

every user with a orgRole including `org.activity.read` can do that

> * [ ]  Usage alerts -- **owners and admin?**

for reading the alerts:  every user with a orgRole including `org.clusters.alerts.read` can do that
for editing/creating alerts: every user with a orgRole including `org.clusters.alerts.create/update` can do that

> * [ ]  Advanced search -- **all users?**

no limitations

is this even an advertised feature? why even mention it?

> * [ ]  Switch organization -- **all users?**

no limitations (if you have >1 org of course)

> * [ ]  Connect your IDP with Camunda -- **not sure here**

i think we are only offering this to enterprise and professional users

> * [ ]  Delete your Camunda account -- **owner, analyst, admin, and task user?**

everyone who is not owning a paid org


i really suggest not to be too specific about this, as this would imply that all further changes to orgrole2org-permissions would need to be relfected on the docs as well (and for sure i do not volunteer to take this responsibility).

**Comment by christinaausley** [comment]
Thank you @ultraschuppi -- we can keep this agnostic and adjust moving forward if we find feedback that users are confused.

Regarding advanced search, I'll make a note in ask-console to see if there are any strong opinions here. If not, and if this isn't an advertised feature, perhaps it should be removed.

TL;DR I'll backport this to the current version of docs and will just need final approval to merge 👍

File: docs/components/console/manage-organization/organization-settings.md [hunk 6]
```diff
@@ -8,15 +8,15 @@ Organization management can be accessed via the menu in the navigation bar.
 
 ![avatar-menue](./img/avatar-menue.png)
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/console/manage-organization/organization-settings.md:9]: Adjusted in latest commit 👍

**Review by akeller — APPROVED** [review]
Approved, under the assumption that we will iterate in the future if we get feedback that the roles/permissions need to be documented more specifically. This can be a huge painpoint in SaaS products if you see documentation for a feature that you cannot access because you don't have the right role/permissions. If the code is not publicly available, they cannot use it as a reference point.

Re: advanced search, this was added late Aug - https://github.com/camunda/camunda-docs/pull/2469. We can look into page views and clicks (if the search bar supports product analytics, I'm unsure). You could argue this is over-documenting, but I could also see how this docs page helps promote the availability of this feature of the console.

**Comment by christinaausley** [comment]
@akeller I'm going to open an issue here for the backlog to see how we're referring to roles across teams. When do we reference them, if not always for clarity? Where are these referenced? How do teams manage updates?

As far as advanced search goes, see https://camunda.slack.com/archives/CKZK2E7RP/p1699454014779609 for additional discussion. @theburi Will keep an eye on this doc, but worth keeping around for now.

**Comment by akeller** [comment]
> @akeller I'm going to open an issue here for the backlog to see how we're referring to roles across teams. When do we reference them, if not always for clarity? Where are these referenced? How do teams manage updates?

@christinaausley I see this as related to https://github.com/camunda/camunda-docs/issues/2046. My understanding is Identity and Console handle all roles and permissions, it's not something individual component teams decide. I will escalate this to get EM & PM support moving forward.
